platform/android/app/src/main/kotlin/coop/polypoly/polypod/FeatureContainer.kt

```diff
@@ -142,7 +142,7 @@ class FeatureContainer(context: Context, attrs: AttributeSet? = null) :
         // Chrome/18.0.1025.133 Mobile Safari/535.19"""
         val userAgentString = webView.settings.userAgentString
 
-        val regex = """(Chrome)\/(?<major>\d+)[\d\.]""".toRegex()
+        val regex = """(Chrome)/(?<major>\d+)[\d.]""".toRegex()
```

Comment:
Are you sure you don't need to escape the dot? `.` in a regex is any codepoint, right?

Reply:
It's redundant character escape '\.' in `RegExp` - inside the square brackets of a character class, many escapes are unnecessary that would be necessary outside of a character class.